contrib/CMakeLists.txt

```diff
@@ -345,76 +350,3 @@ endif()
 if (USE_S2_GEOMETRY)
     add_subdirectory(s2geometry-cmake)
 endif()
-
-# Put all targets defined here and in subdirectories under "contrib/<immediate-subdir>" folders in GUI-based IDEs.
```

Comment:
This fails on "cannot set property on ALIAS target", CC @traceon
As we don't use X-Code, I temporarily removed it.

Reply:
OK, I'll submit adjustments/fixes, after this change is merged.